test/console_web/controllers/compliance_controller_test.exs

```diff
@@ -14,33 +14,23 @@ defmodule ConsoleWeb.ComplianceControllerTest do
 
       insert(:policy_binding, policy_id: cluster.write_policy_id, user_id: user2.id)
 
-      response = conn
+      clusters_content = Console.Compliance.Datasource.Clusters.stream()
```

Comment:
this shouldn't be in a controller test, make it in a separate test, probably under `test/console/compliance` (the test files should match the code file structure).